web/src/components/dashboard/SidebarCardRegisteredForTaxesNudge.tsx

```diff
@@ -25,26 +23,16 @@ export const SidebarCardRegisteredForTaxesNudge = (props: Props): ReactElement =
     }
     queueUpdateTaskProgress(taxTaskId, "COMPLETED");
     await updateQueue.update();
-    router.push("/dashboard");
-  };
-
-  const onClick = (): void => {
-    setModalOpen(true);
-    analytics.event.tax_registration_nudge_button.click.show_tax_registration_modal();
```

Comment:
[Boulder] This is the only place in code where we call this event. Let's remove it from `analytics.ts` and `analytics-legacy.ts`.